src/main/java/roomescape/controller/ReservationTimeController.java

```diff
@@ -13,30 +13,29 @@
 public class ReservationTimeController {
     private final ReservationTimeService reservationTimeService;
 
-    @Autowired
     public ReservationTimeController(ReservationTimeService reservationTimeService) {
         this.reservationTimeService = reservationTimeService;
     }
 
     // 시간 조회
     @GetMapping
-    public ResponseEntity<List<ReservationTime>> getReservationTimes() {
-        List<ReservationTime> reservationTimes = reservationTimeService.getAllReservationTimes();
+    public ResponseEntity<List<ReservationTimeRs>> getReservationTimes() {
+        List<ReservationTimeRs> reservationTimes = reservationTimeService.getAllReservationTimes();
         return ResponseEntity.ok().body(reservationTimes);
     }
 
     // 시간 추가
     @PostMapping
-    public ResponseEntity<ReservationTime> addReservationTime(@RequestBody ReservationTime reservationTime) {
-        reservationTimeService.addReservationTime(reservationTime);
-        return ResponseEntity.ok().body(reservationTime);
+    public ResponseEntity<ReservationTimeRs> addReservationTime(@RequestBody ReservationTimeRq reservationTimeRq) {
+        ReservationTimeRs reservationTimeRs = reservationTimeService.addReservationTime(reservationTimeRq);
+        return ResponseEntity.created(java.net.URI.create("/times/" + reservationTimeRs.getId())).body(reservationTimeRs);
```

Comment:
```suggestion
        return ResponseEntity.created(URI.create("/times/" + reservationTimeRs.getId())).body(reservationTimeRs);
```
위에 import 문을 선언하여 간결하게 작성할 수 있겠네요 :)